packages/gatsby-cli/src/reporter/redux/actions.ts

```diff
@@ -329,4 +341,4 @@ const actions = {
   },
 }
 
-module.exports = bindActionCreators(actions, dispatch)
+module.exports = bindActionCreators<any, any>(actions, dispatch)
```

Comment:
can we change this to `export { ...bindActionCreators<any, any>(actions, dispatch) }`

Also, instead of any, any, i think those are just expecting the actions so maybe try `bindActionCreators<typeof actions>`